.gitignore

```diff
@@ -11,16 +12,21 @@
 *.swp
 *.vcxproj.user
 *.zip
+*/bin
 .DS_Store
 .classpath
 .gradle/
 .idea
 .project
 .settings
 .svn
+.vscode/
 build/
 dependency-reduced-pom.xml
+log/
 out/
 target
 tony-final.xml
+tony-history-server/conf/application.conf
```

Comment:
why we need to specifically ignore this file ? not conf/ or other files?

Reply:
Because the file contains LI HDFS credentials.

Reply:
ah ok